src/store/modules/swap/ethBancor.ts

```diff
@@ -3596,56 +3596,76 @@ export class EthBancorModule
     ];
   }
 
-  @action async getAvailableSpace({
+  @action async getAvailableAndAmountToGetSpace({
     poolId
   }: {
     poolId: string;
-  }): Promise<string> {
+  }) {
```

Comment:
Lets throw in an explicit return type here so it's nice and neat. 
```suggestion
  }): Promise<{ whatever: string }> {
```